### ISSUE TITLE
General purpose methods could be moved to release-common-lib if needed by other release steps

### ISSUE THREAD

**Issue by jweiser** [reactome/data-release-pipeline#49]
In data-exporter/src/main/java/DataExportUtilities.java, there are general purpose methods including:

- splitSet (takes a set of Strings and splits it into the desired number of subsets)
- deleteAndCreateFile (deletes and re-creates an empty file at a given path)
- appendWithNewLine (appends a String or Set of Strings to a file with a new line for each String)
- createXMLDocument (creates a new W3C document object for a standalone XML file)
- attachRootElement (creates a new W3C element object and attaches it as the root element to a W3C document object)
- getElement (creates a new W3C element object with a specified name and text node value)
- transformDocumentToXMLString (returns a String representing a W3C document object as a standalone XML document)

**Comment by SolomonShorser-OICR** [comment]
It seems that these can be categorized as 3 separate utility package "General" (splitSet, appendWithNewLine), "Files" (deleteAndCreateFile), "XML" (the rest).

**Comment by jweiser** [comment]
Yup, that's right.  There could be more going forward in these and other categories, too.
